### PR TITLE
Page Header

### PR DESCRIPTION
**Pull request by patryknow**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -6,3 +6,55 @@
   color: #2c3e50;
   margin-top: 60px;
 }
+
```
Comment (mcwiekala) [inline src/style.css:9]: please move to header.css

File: src/style.css [hunk 2]
```diff
@@ -6,3 +6,55 @@
...
+/* Header */
+.header {
+  background-color: #23272B;
+  display: flex;
+  max-width: 1920px;
+  padding: 40px;
+  align-items: center;
+}
+
+.header__logo {
```
Comment (mcwiekala) [inline src/style.css:19]: good naming convention ;)

File: src/header.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    return pathname[1] === path;
+  }
+
+  connectedCallback() {
+    this.innerHTML = `
+      <header class="header">
+        <img class="header__logo" src="logo.svg" height="60">
+        <nav class="header__nav">
+          <ul class="header__nav-list">
+            ${this.links.map(link => `<li class="header__nav-item ${link.active ? 'header__nav-item--active' : ''}"><a class="header__nav-link" href="${link.href}">${link.name}</li>`).join("")}
```
Comment (mcwiekala) [inline src/header.js:36]: can we format it to more readable way?

File: src/style.css (hunk 1 above)
Comment (patryknow) [reply src/style.css:20]: Done

File: src/header.js (hunk 3 above)
Comment (patryknow) [reply src/header.js]: This is done as well :)

**Pull request merged**